step-templates/octopus-get-octopus-usage.json

```diff
@@ -3,14 +3,14 @@
jectCounts.TargetCount - $ObjectCounts.ActiveCloudRegions - $ObjectCounts.DisabledTargets    \n}\n\n\n$text = @\"\nThe item counts are as follows:\n`tLicensed Target Count: $($ObjectCounts.LicensedTargetCount) (these are active targets de-duped across the instance if running a modern version of Octopus)\n`tProject Count: $($ObjectCounts.ProjectCount)\n`tTenant Count: $($ObjectCounts.TenantCount)\n`tAgent Counts: $($ObjectCounts.WindowsLinuxAgentCount)\n`tDeployment Target Count: $($ObjectCounts.TargetCount)\n`t`tActive and Available Targets: $($ObjectCounts.ActiveTargetCount)\n`t`tActive but Unavailable Targets: $($ObjectCounts.UnavailableTargetCount)\n`t`tActive Target Breakdown\n`t`t`tListening Tentacle Target Count: $($ObjectCounts.ActiveListeningTentacleTargets)\n`t`t`tPolling Tentacle Target Count: $($ObjectCounts.ActivePollingTentacleTargets)\n`t`t`tSSH Target Count: $($ObjectCounts.ActiveSshTargets)\n`t`t`tKubernetes Target Count: $($ObjectCounts.ActiveKubernetesCount)\n`t`t`tAzure Web App Target Count: $($ObjectCounts.ActiveAzureWebAppCount)\n`t`t`tAzure Service Fabric Cluster Target Count: $($ObjectCounts.ActiveAzureServiceFabricCount)\n`t`t`tAzure (Legacy) Cloud Service Target Count: $($ObjectCounts.ActiveAzureCloudServiceCount)\n`t`t`tAWS ECS Cluster Target Count: $($ObjectCounts.ActiveECSClusterCount)\n`t`t`tOffline Target Count: $($ObjectCounts.ActiveOfflineDropCount)\n`t`t`tCloud Region Target Count: $($ObjectCounts.ActiveCloudRegions)\n`t`t`tFtp Target Count: $($ObjectCounts.ActiveFtpTargets)\n`t`tDisabled Targets Targets: $($ObjectCounts.DisabledTargets)\n`t`tDisabled Target Breakdown\n`t`t`tListening Tentacle Target Count: $($ObjectCounts.DisabledListeningTentacleTargets)\n`t`t`tPolling Tentacle Target Count: $($ObjectCounts.DisabledPollingTentacleTargets)\n`t`t`tSSH Target Count: $($ObjectCounts.DisabledSshTargets)\n`t`t`tKubernetes Target Count: $($ObjectCounts.DisabledKubernetesCount)\n`t`t`tAzure Web App Target Count: $($ObjectCounts.DisabledAzureWebAppCount)\n`t`t`tAzure Service Fabric Cluster Target Count: $($ObjectCounts.DisabledAzureServiceFabricCount)\n`t`t`tAzure (Legacy) Cloud Service Target Count: $($ObjectCounts.DisabledAzureCloudServiceCount)\n`t`t`tAWS ECS Cluster Target Count: $($ObjectCounts.DisabledECSClusterCount)\n`t`t`tOffline Target Count: $($ObjectCounts.DisabledOfflineDropCount)\n`t`t`tCloud Region Target Count: $($ObjectCounts.DisabledCloudRegions)\n`t`t`tFtp Target Count: $($ObjectCounts.DisabledFtpTargets)\n`tWorker Count: $($ObjectCounts.WorkerCount)\n`t`tActive Workers: $($ObjectCounts.ActiveWorkerCount)\n`t`tUnavailable Workers: $($ObjectCounts.UnavailableWorkerCount)\n`t`tWorker Breakdown\n`t`t`tListening Tentacle Target Count: $($ObjectCounts.ListeningTentacleWorkers)\n`t`t`tPolling Tentacle Target Count: $($ObjectCounts.PollingTentacleWorkers)\n`t`t`tSSH Target Count: $($ObjectCounts.SshWorkers)\n\"@\n\nWrite-Host $text\n$text > octopus-usage.txt\nNew-OctopusArtifact \"$($PWD)/octopus-usage.txt\""
```

Comment:
The variable name `ActiveListeningTentacleTargets` seems like a contradiction. It's not part of this PR, but looking at the code I think the name of this variable should have been `PassiveListeningTentacleTargets`. Or maybe even just `ListeningTentacleTargets`, as `Active` and `Passive` here are synonyms of `Polling` and `Listening`.